plugins/woocommerce/tests/php/src/Internal/Caches/OrdersVersionStringInvalidatorTest.php

```
@@ -0,0 +1,419 @@
+		$version_after = $this->version_generator->get_version( "order_{$order_id}", false );
+		$this->assertNull( $version_after, 'Version string should be deleted after order update' );
+	}
+
+	/**
+	 * @testdox Changing order customer invalidates the orders list.
+	 */
+	public function test_order_customer_change_invalidates_list(): void {
+		$this->get_invalidator_with_hooks_enabled();
+
+		$order    = \WC_Helper_Order::create_order( 1 );
+		$order_id = $order->get_id();
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$order_id'. (undefined)

(UnusedLocalVariable)